scripts/exregional_run_post.sh

```diff
@@ -276,7 +276,10 @@ bgsfc=${postprd_dir}/${NET}.t${cyc}z.bgsfcf${fhr}.${tmmark}.grib2
 mv_vrfy BGDAWP.GrbF${post_fhr} ${bgdawp}
 mv_vrfy BGRD3D.GrbF${post_fhr} ${bgrd3d}
 # small subset of surface fields for testbed and internal use
-wgrib2 -match "APCP|parmcat=16 parm=196|PRATE" ${bgrd3d} -grib ${bgsfc}
+#wgrib2 -match "APCP|parmcat=16 parm=196|PRATE" ${bgrd3d} -grib ${bgsfc}
```

Comment:
Since the commented line is in the history of the repo, would you mind removing?

Reply:
@christinaholtNOAA sorry just merged and deleted the branch before I saw your comment. Can we save for the next PR?  We'll have to change all of this after HWT anyway

Reply:
Sure. :)